File: data/Gill/siteInfo.m
% site specific script to load site information

% enter orientation of sonics.  Sonic order: tables sorted alphabetically followed by columns sorted in descending order
info.sonicOrientation = [36]; %UU1

% enter manufacturer of SAT.  1 for Campbell, 0 for RMYoung.  RMYoung v =
% Campbell u!; 3 for Gill WindmasterPro
info.sonicManufact = [2];  

% enter orientation of tower relative to sonic head
info.tower = 210; % FM 

% tower elevation
info.siteElevation = 1980; % (m)

% site slope angle
info.angle = 8.2; % mean of 20 m radius buffer

% enter expected table names.  Missing tables will be filled with NaNs to create consistency 
% when multiple output files are concatenated with getData.m
info.tableNames = {'FMDOL_10Hz'}; % modified by Diane

% enter table scan frequencies corresponding to tableNames
info.tableScanFrequency = [10];  %[Hz]

% enter number of columns in each .csv table.  Note that the number of columns in the output structure will 
% be 3 less than the number in the .csv file.  This is because the 4 column date vector is replaced with a Matlab's 
% single-column serial time.  Also, note that View Pro frequently cuts of column 1 (the year!) of the .csv file. 
info.tableNumberOfColumns = [21]; % modified by Diane (21 = +Temp_6.35, RH_6.35)

File: utespac/site_info.py
"""siteInfo – template for site-specific configuration (French Meadows / UU1).

Copy this file to ``<siteFolder>/siteInfo.py`` and adjust the values.
The main script executes it with ``exec()`` and reads the variables listed below.
"""

# Sonic orientations (azimuth from north, degrees).
# Order: tables sorted alphabetically, then columns in ascending height order.
sonicOrientation = [247, 120, 119]  # UU1

# Sonic manufacturer code per sonic:
#   1 = Campbell (CSAT3 / IRGASON)
#   0 = RMYoung  (v_RMY = u_Campbell !)
#   2 = Gill WindmasterPro
sonicManufact = [1, 1, 1]

# Orientation of the tower body relative to true north [degrees].
tower = 210

# Site elevation above sea level [m].
siteElevation = 1980

# Slope angle [degrees] – used for SNSP heat flux decomposition.
angle = 0.0

# Expected table names (must match the CSV filename table identifier).
tableNames = ["FM_20Hz_FluxTower"]

# Sampling frequency for each table [Hz].
tableScanFrequency = [20]

# Number of columns expected in each raw CSV table (before date compression).
tableNumberOfColumns = [19]

# Use local mean reference temperature from slow sensors (HMP) for virtual temperature.
useTrefHMP = True

# Averaging period [min] used to match fast (sonic) data to slow (HMP) data.
avgSlowFreq = 1

# Set True only when running a single high sonic (e.g. siteGill at 51.5 m) but needing
# virtual theta / humidity referenced to the lowest sonic on the full tower.
shiftzRef = False

# Override zRef value [m]; only used when shiftzRef = True.
zRefLowestSon = 4.42

# True if sonic heights in the input data table are in ascending order (low → high).
# Set False if heights are stored high → low (Diane's French Meadows tables).
ascending = True

# Sub-period length [min] for the SSITC steady-state test.  Must evenly divide avgPer.
SSITC_subAvgMin = 5

# Zero-plane displacement height [m] for SSITC stability parameter z/L.
displacementHeight = 0

# Canopy height [m]; used by the Rannik et al. in-canopy σ_w/u* ITC model.
# Set to float('nan') to disable the canopy branch.
canopyHeight = float('nan')

# If True and z ≤ canopyHeight, use the Rannik et al. canopy ITC model instead of
# the Foken above-canopy parameterization.
useCanopyITC = True
